test/Microsoft.AspNet.Mvc.FunctionalTests/LoggingTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if ASPNET50  // json won't serialize in CoreCLR
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LoggingWebSite;
+using LoggingWebSite.Controllers;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Mvc.ApplicationModels;
+using Microsoft.AspNet.Mvc.Logging;
+using Microsoft.AspNet.TestHost;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class LoggingTests
+    {
+        private readonly IServiceProvider _services = TestHelper.CreateServices("LoggingWebSite");
+        private readonly Action<IApplicationBuilder> _app = new LoggingWebSite.Startup().Configure;
+        
+        [Fact]
+        public async Task AssemblyValues_LoggedAtStartup()
+        {
+            // Arrange and Act
+            var logs = await GetLogsForStartup();
```

Comment:
Make this function generic and take the type of log record that you're looking for.